src/components/Card/Root.tsx

```diff
@@ -2,7 +2,7 @@ import { Box, Sprinkles } from "@saleor/macaw-ui-next";
 import React from "react";
 
 export const Root: React.FC<Sprinkles> = ({ children, ...rest }) => (
-  <Box display="flex" flexDirection="column" gap={5} {...rest}>
+  <Box display="flex" flexDirection="column" gap={5} {...rest} opacity="1">
```

Comment:
qustion: why we do not allow to overide opacity,  and why opacity can not by set as props here?

Reply:
That's a mistake, will fix 🫡